### ISSUE TITLE
Pre-trained models

### ISSUE THREAD

**Issue by henryyuanheng-wang** [hasanirtiza/Pedestron#8]
Hi, Thanks for the amazing work! The paper and repo has been super helpful. 

One question I have is regarding the available pre-trained models. For these models, did you pre-train them on CrowdHuman or Wider Pedestrian, and tuned the models for corresponding target dataset (CityPersons, CalTech and ECP)? Is there anyway you can provide the pre-trained models before the fine-tuning step for more general purpose human detection? 

Thank you and looking forward to your reply!

**Comment by hasanirtiza** [comment]
Yes, for Pedestron the models we provide are pre-trained mostly on Wider Pedestrian and subsequently fine-tune on the target set as discussed in the paper. We can definitely provide models without fine-tuning on the target set for general purpose human detection very soon.

**Comment by hasanirtiza** [comment]
You can access the Cascade Mask R-CNN model for generic human detection at:
https://drive.google.com/open?id=1rXopG04Dv-HKge3ZyqNYAHdCi9gmLuNe

Also updated in the [README.md](https://github.com/hasanirtiza/Pedestron/blob/master/README.md).

**Comment by henryyuanheng-wang** [comment]
Thank you so much man! This is super helpful. 💯 Good luck with research!
